www/assets/js/components/SearchFilterDrawer.tsx

```diff
@@ -61,6 +65,9 @@ export default function SearchFilterDrawer(props: Props) {
       <div className="contents">
         <FacetDisplay {...props} />
       </div>
+      <div className="col-12 px-5 mt-3 pb-5">
+        <Footer />
+      </div>
```

Comment:
**Suggestion:** I think we should use a `<footer />` tag here. *Or in the react component, though hugo templates seem where our "structure" is, so here seems better to me. In which case, maybe change the react component to FooterContent or something.*

(Leaving this as a suggestion because I am not entirely sure the consequences of using footer vs something else, but footer for footers seems good.)